stylesheets.css

```diff
@@ -28,9 +29,20 @@ body {
     
 }
 .div1 {
-    background: lightblue;
+    background: lightskyblue;
     height: 3rem;
     display: grid;
     align-items: center;
     box-shadow: green;
+    width: 100%; position: fixed; top: 0;
+    
```

Comment:
Good try. If you noticed your menu wasn't fixed to the top. This is because you have 3 properties on the same line. 
To get the menu fixed, try putting each property-value pair on a separate line. Here is a screenshot: https://d.pr/u5oHT0. Once done, add, commit and push to the same branch.

Reply:
Ok, thanks!